src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/ApprovalControllerTests.java

```diff
@@ -12,13 +14,17 @@
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
 import org.springframework.web.servlet.view.UrlBasedViewResolver;
 import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.smallfull.Approval;
 import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
```

Comment:
Can you remove the `uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet` as this is redundant here?